README.md

```diff
@@ -36,72 +38,117 @@
 - [Contributing](#contributing)
 - [License](#license)
 
-
 ## About Mitt Helsingborg
 
-
 ### Built With
 
-* [React Native](https://reactnative.dev/)
-* [TypeScript](https://www.typescriptlang.org/)
-* [Styled Components](https://styled-components.com/)
-
+- [React Native](https://reactnative.dev/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [Styled Components](https://styled-components.com/)
 
 ## Getting Started
 
 To get a local copy up and running follow these simple steps.
 
-
-
 ### Prerequisites
 
 - [Node.js and NPM](https://nodejs.org/en/download/package-manager/)
 - [React Native Development Environment](https://reactnative.dev/docs/environment-setup/)
 - [Eslint](https://helsingborg-stad.github.io/dev-guide/docs/development/linters/eslint.html)
 
-
 ### Installation
 
 1. Clone the repo
+
 ```sh
 git clone https://github.com/helsingborg-stad/app-mitt-helsingborg.git
 ```
+
 2. Install NPM packages
+
 ```sh
 npm install
 ```
+
 3. Run app on iOS simulator
+
 ```sh
 react-native run-ios
 ```
+
 4. Run app on Android emulator
+
 ```sh
 react-native run-android
 ```
 
-#### Component library (Storybook)
-1) Set env variabel IS_STORYBOOK to true in .env
-2) Launch storybook server by running command "npm run storybook" or "yarn storybook"
-3) Launch application in simulator by running command "react-native run-ios".
-4) Now you should se storybook running in the simulator.
+## Setup environment variables
+
+### Backend selector
+
+Using the file .env in the root of the project, various parameters could be used to
```

Comment:
I would consider it best practices to surround file names, variables etc. with back-ticks so they are highlighted better 🙂.

.env -> `.env`
APP_ENV -> `APP_ENV`
API_ENVS -> `API_ENVS`

But if it's too many repeating occurrences feel free to skip as otherwise it might get messy to read instead.

Reply:
fixed